src/etc/inc/filter.lib.inc

```diff
@@ -201,90 +201,87 @@ function filter_core_get_default_nat_outbound_networks()
 function filter_core_rules_system($fw, $defaults)
 {
     global $config;
+    $ipv6_allow = isset($config['system']['ipv6allow']);
 
     $dhcrelay6_interfaces = plugins_run('dhcrelay_interfaces', ['inet6']);
     $dhcrelay6_interfaces = !empty($dhcrelay6_interfaces['dhcrelay']) ? $dhcrelay6_interfaces['dhcrelay'] : [];
 
     // block All IPv6 except loopback traffic
```

Comment:
```suggestion
```

it's obvious now :)

Reply:
hehe, right https://github.com/opnsense/core/pull/8010/commits/a1138e57f4b4fb9db8d6b5745e95b2f449572b52